src/FlashMessages.php

```diff
@@ -161,11 +165,17 @@ public function prepareMessages(SessionInterface $session, string $sessionKey):
             return;
         }
 
-        $sessionMessages = $session->get($sessionKey);
-        $sessionMessages = ! is_array($sessionMessages) ? [] : $sessionMessages;
+        $sessionMessages = $this->getStoredMessages($sessionKey);
 
+        /** @var array<string,mixed> $currentMessages */
         $currentMessages = [];
         foreach ($sessionMessages as $key => $data) {
+            /**
+             * The public API of flash() and flashNow() explicitly allow calling
+             * code to pass `$value`s of any time, making this unavoidable.
```

Comment:
guess you mean `type` rather than `time`? 😄

Reply:
Indeed.